modules/apps/users-admin/users-admin-impl/src/main/java/com/liferay/users/admin/internal/search/UserBulkReindexer.java

```diff
@@ -31,6 +41,51 @@ public class UserBulkReindexer implements BulkReindexer {
 
 	@Override
 	public void reindex(long companyId, Collection<Long> classPKs) {
+		if (_databaseInMaxParameters == Integer.MAX_VALUE) {
+			_reindex(companyId, classPKs);
+
+			return;
+		}
+
+		List<Long> classPKsList = ListUtil.fromCollection(classPKs);
```

Comment:
@marianoalvarosaiz, the condition is not enough. When _databaseInMaxParameters is not Integer.MAX_VALUE. If classPKs.size() <= _databaseInMaxParameters, it also should directly invoke `_reindex(companyId, classPKs);`